packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -61,6 +61,8 @@ export interface MetricDescriptor {
   readonly unit: string;
   /** MetricDescriptor type */
   readonly type: MetricDescriptorType;
+  /** Does this metric only count up */
```

Comment:
is this a question ?

Reply:
It is an explanation of the flag. I'll change it to be a statement